runtime/gpufortrt/include/gpufortrt_api.h

```diff
@@ -105,6 +109,16 @@ extern "C" {
           int async_arg,
           bool never_deallocate);
 
+// Same as copyin?
```

Comment:
If I remove gpufortrt_copy and gpufortrt_copy_asyn, I get the following errors:

undefined reference to `gpufortrt_copy' 
undefined reference to `gpufortrt_copy_asyn' 

which happens for:

!$acc data copy(x,y(1:2*n))

Reply:
The missing reference error stems from the following erroneous jinja macros, which
wrongly generate an interface and specialized callbacks for the "copy" mapping.
If you remove the `"copy"` in both cases from the list, this should fix this error.

```
runtime/gpufortrt/src/gpufortrt_api.macros.f90:172:{% macro render_basic_copy_routines() %}
runtime/gpufortrt/src/gpufortrt_api.macros.f90-173-{#######################################################################################}
runtime/gpufortrt/src/gpufortrt_api.macros.f90-174-{% for clause in ["create","copyin","copy"] %}
--
runtime/gpufortrt/src/gpufortrt_api.macros.f90:228:{% macro render_specialized_copy_routines(datatypes,max_rank) %}
runtime/gpufortrt/src/gpufortrt_api.macros.f90-229-{#######################################################################################}
runtime/gpufortrt/src/gpufortrt_api.macros.f90-230-{% for clause in ["create","copyin","copy"] %}
```

Reply:
Fixed in commit https://github.com/ROCmSoftwarePlatform/gpufort/pull/45/commits/170632555085be546805a73e8eb2eef324355519

Reply:
I should remove copy from the following line too:

345: {% for clause in ["create","copyin","copy"] %}

Otherwise, it still generated the wrong interfaces. Now fixed in commit https://github.com/ROCmSoftwarePlatform/gpufort/pull/45/commits/170632555085be546805a73e8eb2eef324355519